Code/Editor/EditorViewportSettings.cpp

```diff
@@ -627,4 +634,166 @@ namespace SandboxEditor
         AzToolsFramework::SetRegistry(CameraFovSetting, aznumeric_cast<double>(fovDegrees));
     }
 
+    void ResetCameraSpeedScale()
```

Comment:
this is a lot of boilerplate repetitive code.

Reply:
Right, but there isn't a good alternative right now... This is useful for customers and is clear. Yes it could be streamlined but it's not actually a huge complexity add. Do you have an alternative idea to share?

Reply:
I have some ideas with using traits.

Reply:
I have a couple basic thoughts nothing I can show though. want something similar to CVAR so maybe some kind of trait interface maybe?

Reply:
I agree that would be cool, and something we can discuss in future but it's out of scope of this change unfortunately